package-scripts.js

```diff
@@ -185,6 +185,10 @@ module.exports = {
         esm: {
           script: 'MOCHA_TEST=esm nps test.browser.unit',
           description: 'Test mocha ESM support'
+        },
+        bundle: {
```

Comment:
this isn't really a _browser_ test because we're not invoking Karma.  maybe should just move it to be its own thing, e.g. `test.bundle`. also, there's a `test()` function to help.